### PR TITLE
Fixed compile warnings.

### PR DESCRIPTION
**Pull request by jcs090218**
This is the same as #256. I somehow mess up the base of the commitment so I just made another PR.

**Edit:** I also add a TODO for macro `all-the-icons-fileicon`. Not sure how to solve this yet. 😕

### CODE REVIEW

**Comment by jcs090218** [comment]
The CI still fails. I think I will just leave this issue here for now. Still not sure how to solve these undeclare issue. 😕

File: dashboard.el [hunk 1]
```diff
@@ -23,7 +23,6 @@
 ;;; Code:
 
 (require 'seq)
-(require 'page-break-lines)
```
Comment (jcs090218) [inline dashboard.el:26]: `page-break-lines` is autoload. Don't need to require it on top.

**Comment by jcs090218** [comment]
@JesusMtnez Can you give this a look? This PR should be ready to merge! Thanks!

**Review by JesusMtnez — APPROVED** [review]

**Comment by jcs090218** [comment]
@JesusMtnez Thank you! ;)

**Pull request merged**